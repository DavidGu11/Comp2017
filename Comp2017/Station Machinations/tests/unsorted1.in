Sydney::Melbourne::11:01:23
Sydney::Melbourne::23:13:22
Sydney::Melbourne::21:12:22
Sydney::Melbourne::09:38:23
Sydney::Melbourne::10:01:23
Sydney::Melbourne::10:30:23
Sydney::Melbourne::0:59:23