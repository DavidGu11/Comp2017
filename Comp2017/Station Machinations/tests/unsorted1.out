The next train to Melbourne from Sydney departs at 23:13:22
